WordPress/Classes/Utility/ContextManager.h

```diff
@@ -49,7 +49,7 @@ FOUNDATION_EXTERN NSString * const ContextManagerModelNameCurrent;
  @param modelName Model name in Core Data data model file.
                   Use ContextManagerModelNameCurrent for current version, or
                   "WordPress <version>" for specific version.
- @param storeURL Database location.
+ @param storeURL Database location. Use "/dev/null" to create an in memory database.
```

Comment:
Nitpick: 

```suggestion
 @param storeURL Database location. Use "/dev/null" to create an in-memory database.
```

Reference: [Wikipedia](https://en.wikipedia.org/wiki/In-memory_database) or you can google "in memory database".

I'm sure there's an English grammar rule to know when to hyphen like this, but I don't know it. I've sort of developed an hear for it thanks to Grammarly correcting me for the past 2 years 🙃